BubbleSort/BubbleSort.Tests/BubbleSort.Tests.cs

```diff
@@ -0,0 +1,22 @@
+namespace BubbleSort.Tests;
+using BubbleSort;
+public class Tests

```

Comment:
```suggestion
namespace BubbleSort.Tests;

using BubbleSort;

public class Tests
```
И ещё не хватает шапки с информацией о лицензии